datumaro/plugins/camvid_format.py

```diff
@@ -61,6 +61,7 @@ class CamvidPath:
     SEGM_DIR = "annot"
     IMAGE_EXT = '.jpg'
     MASK_EXT = '.png'
+    PATTERN = re.compile(r'(.+[.]\S+) (.+[.]\S+)?')
```

Comment:
Please don't expect input datasets to be generated by Datumaro. Instead, consider meaningful relaxations to the format. Expect and fix little and insignificant errors you can think of.

I think, the pattern can be improved with `r(.+\.\S+)(?:\s+(.+\.\S+)?)\s*`
- to allow any number of whitespaces between entries
- to allow any number of trailing whitespaces (.strip() would also work)
- to allow lines like `a/b/c .d  \n`